models/users.js

```diff
@@ -335,6 +341,16 @@ const fetchUserSkills = async (id) => {
   }
 };
 
+const getIssueAssigneeRdsInfo = async (githubUsername) => {
```

Comment:
NIT: If this function is used in some place to get info based on GitHub userId, calling it `getIssueAssigneeRdsInfo` won't be good, Can we change the name to something more general